src/indigo/less/head.less

```diff
@@ -15,7 +30,16 @@
     font-size: 24px;
     border-bottom: solid 2px #1e2129;
 
-    > a {
```

Comment:
toto bude asi typo, no spravilo to Keboola logo modre (https://keboola.slack.com/archives/C02J3DKH3/p1539719715000100)

Reply:
Jo asi bude. Bylo by fajn asi přidat třídu i k tomu logu a pak to stylovat podle toho. 
Pošlu na to PR teda. Pak by se přidala ta třída a nastavila barva.

Reply:
Respektive se může vrátit toto co tam bylo `> a `  k tomu `.kbc-notification-access` na řádce 38.

Reply:
No, je tam uz nieco ako `.kbc-logo>a`

Reply:
aha, ty vlastne chces dat class aj na ten odkaz ... chapem